src/models/customer.ts

```diff
@@ -1,5 +1,6 @@
 export interface Customer {
```

Comment:
```suggestion
export interface ICustomer {
```

As per company standard, interfaces should be prefixed with `I`.

Reply:
Applied changes